transform/instance_test.go

```diff
@@ -496,7 +512,15 @@ func TestScalarTransform(t *testing.T) {
 		}
 
 		got, err := st.transform(test.in)
+
 		if err != nil {
+			if err.Error() == test.wantError {
+				break // pass
```

Comment:
continue rather then break. It happens to be the last test right now but in the future more could be added later.